### PR TITLE
add delay to panning

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -203,6 +203,7 @@ Site.Shapes = {
 Site.Map = {
   panZoneSize: 150, // in pixels
   panSpeed: 0.04,
+  delay: 100, // in ms
```
Comment (varffs) [inline src/js/main.js]: delay of what tho?

**Review by varffs — CHANGES_REQUESTED** [review]
Works fine but needs better variable name and also I think you should commit built files each time. Weird to have to run gulp on the branch to make it correct

**Comment by mixcas** [comment]
Will fix 

@varffs sorry about the built files, i feel i been committing them. Ill pay more attention next time.

**Comment by mixcas** [comment]
This is ready. Also, @varffs all 3 commits include built files. Might be a problem on your side.

**Review by varffs — APPROVED** [review]

**Pull request merged**